src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -21,6 +21,13 @@ public CommandResult(String feedbackToUser) {
         relevantPersons = null;
     }
 
+    /**
```

Comment:
not necessary to have a header comment which adds no additional value. in general can avoid for accessors